src/backend_model_instance.cc

```diff
@@ -57,6 +57,10 @@
 
 namespace triton { namespace core {
 
+// Condition when a backend thread is shared.
```

Comment:
Update commit: [Use function instead of macro for ShareBackendThread](https://github.com/triton-inference-server/core/pull/189/commits/16d0054d7f5c7e63da636d180a1eeb4af61e1ae9)